test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -110,4 +118,66 @@ describe('Your PSP Stripe page', () => {
     cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
     cy.get('span').contains('Government entity document').should('have.attr', 'href', `/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}/government-entity-document`)
   })
+
+  describe('Bank details task', () => {
+    it('should click bank details task and display bank details page correctly', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.setEncryptedCookies(userExternalId)
+      cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
+      cy.get('span').contains('Bank Details').click()
+      cy.get('#bank-details-form').should('exist')
+        .within(() => {
+          cy.get('input#account-number').should('exist')
+          cy.get('input#sort-code').should('exist')
+          cy.get('button').should('exist')
+          cy.get('button').should('contain', 'Save and continue')
+        })
+    })
+
+    it('should redirect back to the task List when valid bank details submitted', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.get('input#account-number[name="account-number"]').type(accountNumber)
+      cy.get('input#sort-code[name="sort-code"]').type(sortCode)
+      cy.get('#bank-details-form > button').click()
+
+      cy.get('h1').should('contain', 'Your payment service provider (PSP) - Stripe')
+    })
+
+    it('should have Bank details hyperlink removed when complete and status updated to "COMPLETE" ', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: true,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
```

Comment:
Can we get rid of these?

Reply:
This one we need bank account to be true. I can remove the rest of the tasks?